Gem/Code/Source/Weapons/SceneQuery.cpp

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#include <Source/Weapons/SceneQuery.h>
+#include <AzFramework/Physics/Common/PhysicsSimulatedBody.h>
+#include <AzFramework/Physics/ShapeConfiguration.h>
+#include <AzFramework/Physics/PhysicsScene.h>
+#include <AzFramework/Physics/Material.h>
+#include <Multiplayer/NetworkEntity/INetworkEntityManager.h>
+
+namespace MultiplayerSample
+{
+    namespace SceneQuery
+    {
+        static AZStd::shared_ptr<Physics::ShapeConfiguration> GatherShapeToPhysicsShape(const GatherShape& gatherShape)
+        {
+            const float defaultValue = 1.0f; // TODO: Pass via gather params
```

Comment:
So the input param here should ultimately be a GatherParams& from what I gather. Is the size derivable from the filter fed to the ShapeCastRequest? It might make sense to address this now if so.